pages/tag/_query/index.vue

```diff
@@ -1,36 +1,18 @@
 <template>
-	<b-row align-h="center">
-		<div id="container">
+	<div class="container">
 			<h1 id="title">
-				<b>TAG: "{{ searchQuery }}"</b>
+				TAG: "{{ searchQuery }}"
```

Comment:
- Don't put "TAG" in uppercase; if it needs to be uppercased we can do that with CSS.
- Use [curly quotes](https://practicaltypography.com/straight-and-curly-quotes.html) around the search query.